terms.html

```diff
@@ -15,20 +15,21 @@
 A set of one or more <a>claims</a> made by the issuer.
 A <dfn data-lt="verifiable credentials">verifiable credential</dfn>
 is a credential that is tamper-resistant and whose authorship can be
-cryptographically verified.
+cryptographically verified. Credentials can be used to build
+<a>presentations</a>, which can also be cryptographically verified.
   </dd>
   <dd>
 Note: The claims in a credential may be about different subjects.
   </dd>
   <dt><dfn data-lt="credential validation">validation</dfn></dt>
   <dd>
-The process that demonstrates the information in a <a>credential</a> is
-well-formed.
+The process that demonstrates the information in a <a>credential</a>
+or <a>presentation</a> is well-formed.
   </dd>
-  <dt><dfn data-lt="credential verification">verification</dfn></dt>
+  <dt><dfn data-lt="verification">verification</dfn></dt>
   <dd>
 The process that cryptographically demonstrates the authenticity of a
-<a>credential</a>.
+<a>credential</a> or a <a>presentation</a>.
```

Comment:
The original text is wrong... we should say "verifiable credential" and "verifiable presentation"

Reply:
will adjust